### DISCUSSION TITLE
Complier warning for hb-font.cc when using HarfBuzz v7.0.0

### DISCUSSION THREAD

**Discussion by honkar-jdk** [harfbuzz/harfbuzz#4123] [General] [answered]
I'm running into following warning when using harfbuzz version v7.0.0.

libharfbuzz\hb-font.cc(2492): error C2220: the following warning is treated as an error
libharfbuzz\hb-font.cc(2492): warning C4805: '==': unsafe mix of type 'bool' and type 'hb_bool_t' in operation

Can this warning be suppressed without causing any further problems?

**Comment by behdad** [comment, marked as answer]
Duplicate of https://github.com/harfbuzz/harfbuzz/issues/4122

**Reply by honkar-jdk** [reply]
@behdad Thank you for the quick turnaround. Sorry I might have created a issue and discussion at same time, didn't notice that there is an option of creating issue from a discussion.

**Reply by behdad** [reply]
No worries. Thanks.
